docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.three_column_w_image.scss

```diff
@@ -1,3 +1,11 @@
+// An note on the functionality of images
+// --------------------------------------
+// The height and width of images are set and can be changed in layout builder iamge styles.
```

Comment:
iamge -> image

Reply:
and maybe also say instead: layout build image styles -> layout builder configuration for images

Reply:
I like that!! 👍